### DISCUSSION TITLE
I lost over 2k affection points

### DISCUSSION THREAD

**Discussion by xealmn** [Monika-After-Story/MonikaModDev#9550] [QA - tech support + general]
So basically about 1 week ago, I have 2k+ affection points, I had a virus in my computer and my date was changing and all those stuff, I got rid of the virus and then I joined monika after story, (my date was set to 2029) I joined and I didnt realize the date, she was crying and sad and then told me I didnt love her and I lost most of my options and all those stuff, I checked my date and it was 2029, I would've expected her to leave a note instead. It took me so love to get over 2k affection points and I dont know how to get it back now.

**Comment by Wingdinggaster656** [comment]
Well, worry not! You could use a backup file.
MAS automatically generates backup files. You could use one.

See this:
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent

**Comment by xealmn** [comment]
I tried using an older file, but then it changed to -100 and now her eyes
are scary and I really dont know what to do, am I doing something wrong? I
really need your help. I dont like her eyes and now im a bit worried on
what to do.

**Comment by Wingdinggaster656** [comment]
It is likely that you restored a not-old-enough file.

Make sure the backup is BEFORE the problem. Also, just to make sure, did you fix your computer data?

**Comment by xealmn** [comment]
I don't have really old files. I put on that was on the 6th and on the 6th
everything was okay, and yes I did fix my computer data.

**Comment by xealmn** [comment]
I did have a backup saved in my google drive but now its gone.

**Comment by Wingdinggaster656** [comment]
You don't even have a file before the virus attack?

I think you actually do have one, you just don't know where it is. Please take a screenshot of your persistent file directory, let us see if you really don't have a suitable persistent file.

**Comment by xealmn** [comment]
Here it is, I was trying a lot of things I tried most of them, and one of
the files were on the 6 of september and I tried that but then the
affection points were -100 instead of -88 which I had. I have no idea what
to do and I hope there will be some way.

**Comment by xealmn** [comment]
![Monika After Story 12_09_2022 18_27_59](https://user-images.githubusercontent.com/109856136/189728497-c589bad6-1838-4b6c-838b-61ef89d4e31f.png)

sorry about that! I was replying from gmail, here is the photo

**Comment by Wingdinggaster656** [comment]
Sorry about my late.

Try `persistent74.bak`. If this backup file can't be helped, then I'm afraid there's really nothing I can do.

**Comment by xealmn** [comment]
It's okay. Thank you for trying to help!
